### ISSUE TITLE
UltraPermissions Support

### ISSUE THREAD

**Issue by BlackGamer000** [darbyjack/Guilds-Plugin#89]
Hello! 

Since in the config there is this: 

`# It will allow 16 characters before the user's Username.
hooks:
  worldguard: true
  nametagedit: false
  serverlistplus: false

# This allows you to configure how nametags will work in game with NameTagEdit.
# You can also display a guild's prefix by using {prefix} instead of {guild}.
nametagedit:
  prefix:
    enabled: true
    prefix: "&8[&b{guild}&7]&r "
  suffix:
    enabled: false
    suffix: "&7[&b{guild}&7]&r "`

I'd like to suggest the addon of the UltraPermissions plugin, that is a [GUI Permission plugin](https://www.spigotmc.org/resources/ultra-permissions.42678/)

Basically recognize Prefixes and Suffixes automatically.

Thanks a lot and good bye c:

**Comment by darbyjack** [comment]
Couple things,

I don't really understand what you mean by this. You can just modify the config and then reload the plugins config and you're good, no need for a GUI. I talked to the Dev of UP and he agrees what you're asking doesn't make sense.

Feel free to elaborate.
